packages/office-ui-fabric-react/src/components/Button/Button.tsx

```diff
@@ -13,9 +13,9 @@ import { PrimaryButton } from './PrimaryButton/PrimaryButton';
  */
 export class Button extends BaseComponent<IButtonProps, {}> {
   /**
-   * Set this BaseComponent._resolveComponentRef to false, bypassing resolution of componentRef.
+   * Set this BaseComponent._skipComponentRefResolution to true, bypassing resolution of componentRef.
    */
-  protected _shouldUpdateComponentRef = false;
+  protected _skipComponentRefResolution = true;
```

Comment:
Technically, this could be considered a breaking change, as it's `protected`.

Reply:
I changed `BaseComponent.resolveComponentRef` to `skipComponentRefResolution`, which defaults to `false` and is far more clear about how the override works. (Previously it seemed like there was a race condition where the base class would initialize the variable in its constructor, after super, which meant that a subclass initializing the value in its constructor would get overwritten.